lib/src/common/timeouts.dart

```diff
@@ -0,0 +1,22 @@
+class TimeoutValues {
```

Comment:
@jakemac53 @lrhn - Would you have any concerns about using a record type with named fields in place of this class? The package is already on language version 3.

Reply:
No concerns. It means there won't be a nice `toString`, possibly it will lack the names in production.
But the  `toString` is for debugging only (there is no `parse`), so that shouldn't be a problem.

I wouldn't introduce a typedef then, just inline the type in the function declaration:
`({Duration script, Duration implicit, Duration plageLoad})`,
as a multi-value result.

Reply:
I wonder if we need to worry about this interface changing. Without the class it could be harder to add more fields, because the record types are unrelated once we add a new named field. With the class (if we make it `final class`) we can add a new field as long at is isn't `required` in the constructor.